pkg/kv/kvserver/kvstorage/init.go

```diff
@@ -291,46 +293,93 @@ func IterateRangeDescriptorsFromDisk(
 	return err
 }
 
-type EngineReplicas struct {
-	Uninitialized map[storage.FullReplicaID]struct{}
-	Initialized   map[storage.FullReplicaID]*roachpb.RangeDescriptor
+// LoadedReplicas represents the Replicas present on a storage engine.
+type LoadedReplicas struct {
+	Uninitialized ReplicaMap
+	Initialized   ReplicaMap
 }
```

Comment:
IMO, this type is not needed, one can return `ReplicaMap` directly and sift through (un)inited replicas if needed.

Reply:
Done!